pkgs/development/python-modules/ngs-tools/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  joblib,
+  numba,
+  numpy,
+  pysam,
+  shortuuid,
+  tqdm,
+  typing-extensions,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "ngs-tools";
+  version = "1.8.5";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "OA4jahAcWxrDwPzbzJCKIQF5tu8qk/vqn06w7C7cHeA=";
```

```suggestion
    hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```
Please use SRI hash instead.